__tests__/workers/cdc/primary.ts

```diff
@@ -3263,3 +3276,161 @@ describe('user streak change', () => {
     ]);
   });
 });
+
+describe('bookmark change', () => {
+  type ObjectType = Bookmark;
+  const base: ChangeObject<ObjectType> = {
+    userId: '1',
+    postId: 'p1',
+    createdAt: new Date().getTime(),
+    listId: null,
+    remindAt: null,
+  };
+
+  describe('on create', () => {
+    it('should not run reminder workflow', async () => {
+      const after: ChangeObject<ObjectType> = base;
+      await expectSuccessfulBackground(
+        worker,
+        mockChangeMessage<ObjectType>({
+          after,
+          before: null,
+          op: 'c',
+          table: 'bookmark',
+        }),
+      );
+      expect(runReminderWorkflow).not.toHaveBeenCalled();
+    });
+
+    it('should run reminder workflow if remind at is present', async () => {
+      const date = new Date();
+      const after: ChangeObject<ObjectType> = {
+        ...base,
+        remindAt: date.getTime(),
+      };
+      await expectSuccessfulBackground(
+        worker,
+        mockChangeMessage<ObjectType>({
+          after,
+          before: null,
+          op: 'c',
+          table: 'bookmark',
+        }),
+      );
+      expect(runReminderWorkflow).toHaveBeenCalledWith({
+        postId: 'p1',
+        remindAt: date.getTime(),
+        userId: '1',
+      });
+    });
+  });
+
+  describe('on update', () => {
+    it('should NOT cancel reminder workflow if remind at is NOT present before', async () => {
```

Comment:
What does before mean here?

Reply:
I did see before.remindAt and after.remindAt, so just more to understand how we separate those

Reply:
In CDC, there is an object for `before` and object for `after`. That's what we mean here.